dist/tpm2-abrmd.service.in

```diff
@@ -1,12 +1,12 @@
-# copy this file into /etc/systemd/system
+# copy this file into /lib/systemd/system
 [Unit]
 Description=TPM2 Access Broker and Resource Management Daemon
 
 [Service]
 Type=dbus
 Restart=always
 RestartSec=5
-EnvironmentFile=-@sysconfdir@/default/tpm2-abrmd
+EnvironmentFile=-@sysdefaultdir@/tpm2-abrmd
```

Comment:
Is this even installed? I also think this line could just be removed from the systemd unit file.

Reply:
I forgot about this file. I don't know that I had any real plan for it other than having it in place on the off chance that the daemon required some stuff in the `env`, maybe for customizing the options passed to the daemon etc. Just made a ticket to sort this out: #339

Reply:
I will comment it out for now.

Reply:
@dantpm I would just remove it really. If someone wants to later bring it back, there's the git history to dig for it.

Reply:
OK--I will remove it.
If it is readded, please document that --sysconfdir is needed or re-add  --with-systdefaultdir

Reply:
@dantpm yeah, although `--sysconfdir` is a pretty standard autootols variable.